README.md

```diff
@@ -1,20 +1,41 @@
 # Rails Dynamic Request Lab
 
+
 ## Objectives
 
-1. Draw a route with a dynamic variable
-2. Use a routes variable in params within the logic of an action
-3. Assign an instance variable in a controller action
-4. Use a controller instance variable to generate a dynamic ERB template.
-5. Build a classical show action/view
+* Draw a route with a dynamic variable
+
+* Use a routes variable in params within the logic of an action
+
+* Assign an instance variable in a controller action
+
+* Use a controller instance variable to generate a dynamic ERB template.
+
+* Build a classical show action/view
+
+
+## Instructions
+
+There are three RSpec/Capybara specs in this lesson, you can run them with the command ```rspec spec/features/student_spec.rb```, make sure they pass without breaking the other tests, to complete this section all specs should pass.
+
+In this lab you will work through the full MVC pattern, at a high level you will integrate the following scenario in your application:
```

Comment:
Also give them a description of the domain so something like "You are building a student management app for The Flatiron School"